pkg/mountutil/mountutil.go

```diff
@@ -57,6 +57,9 @@ func ProcessFlagV(s string, volStore volumestore.VolumeStore) (*Processed, error
 	)
 
 	split := strings.Split(s, ":")
+	if split[0] == "" {
+		split = split[1:]
+	}
```

Comment:
wrong result  for `nerdctl run  --mount type=volume,dst=/foo,readonly`

Reply:
This does not create for docker as well; however I have made the code changes for this as well.

Reply:
Sorry for going back and forth; adding incompatible changes did not make much sense as the code there was fragile. Keeping it simple

Reply:
FYI:
```
➜  nerdctl git:(anon-volume) ✗ docker run --mount type=volume,dst=/foo,readonly alpine
docker: Error response from daemon: invalid mount config for type "volume": must not set ReadOnly mode when using anonymous volumes.
See 'docker run --help'.
➜  nerdctl git:(anon-volume) ✗ docker run --mount type=volume,dst=/foo,rw alpine      
invalid argument "type=volume,dst=/foo,rw" for "--mount" flag: invalid field 'rw' must be a key=value pair
See 'docker run --help'.
➜  nerdctl git:(anon-volume) ✗ 
```